framework/rbac/BaseManager.php

```diff
@@ -190,6 +190,20 @@ public function getRoles()
     }
 
     /**
+     * Returns defaultRoles as array of Role objects
+     * @since 2.0.12
+     * @return Role[] default roles. The array is indexed by the role names
+     */
+    public function getDefaultRoles()
+    {
+        $result = [];
+        foreach ($this->defaultRoles as $roleName) {
+            $result[$roleName] = new Role(['name' => $roleName]);
```

Comment:
We already have method `createRole()` for this purpose.
I think it is better to call it instead

```php
$result[$roleName] = $this->createRole($roleName);
```